### DISCUSSION TITLE
Generating US electricity baseline with eGRID 2022 data?

### DISCUSSION THREAD

**Discussion by fchichorro** [NETL-RIC/ElectricityLCI#269] [Q&A]
Is it possible to generate eLCI openLCA files with the latest 2022 eGRID data? (I am assuming that the latest is found [here](https://www.epa.gov/egrid/download-data))

**Comment by m-jamieson** [comment]
I think if you follow the guidance in the response to your other issue, you might find some success: [https://github.com/USEPA/ElectricityLCI/issues/268#issuecomment-2460029008](https://github.com/USEPA/ElectricityLCI/issues/268#issuecomment-2460029008)

**Reply by fchichorro** [reply]
Thank you, I will try that suggestion next! But regarding eGRID 2022, I read in the [DataYears.md](https://github.com/USEPA/ElectricityLCI/blob/master/DataYears.md) file that the latest eGRID compatible data year is 2016, is that right?

**Reply by dt-woods** [reply]
We didn't have 2022 data ready in time for v2 release (https://github.com/NETL-RIC/ElectricityLCI/releases/tag/v2.0) &mdash; it uses 2021 eGRID data (as indicated in ELCI_2022_config.yml). Note that v2.1 (in preparation) uses eGRID 2023.
